src/index.tsx

```diff
@@ -8,13 +8,16 @@ import {
   omit,
   onlyNumbers,
   parseOnBlur,
+  parseRangeOnBlur,
   pick,
 } from './utils';
 import { BasicDatePicker, RangeDatePicker } from './pickers';
 import { Locale, SemanticDatepickerProps } from './types';
 import Calendar from './components/calendar';
 import Input from './components/input';
 
+import { legacyParse } from '@date-fns/upgrade/v2';
```

Comment:
Can you move this import to line 2 just for convenience and code organization, please?

Reply:
Import line removed